### PR TITLE
Add support for remoting.

### PR DESCRIPTION
**Pull request by pthatcherg**
***
<a href="https://pr-preview.s3.amazonaws.com/webscreens/openscreenprotocol/pull/173.html" title="Last updated on May 23, 2019, 2:43 AM UTC (76ecd22)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webscreens/openscreenprotocol/173/fb68050...76ecd22.html" title="Last updated on May 23, 2019, 2:43 AM UTC (76ecd22)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -902,13 +906,29 @@ remote-playback-modify-request message with the following values:
 : controls
 :: Updated controls as defined in {#remote-playback-state-and-controls}
 
+: remoting-audio
```
Comment (baylesj) [inline index.bs]: Can this be omitted if we only support video?

File: index.bs [hunk 2]
```diff
@@ -902,13 +906,29 @@ remote-playback-modify-request message with the following values:
...
+:: Audio encodings that the controller can send to the receiver once they
+    are acked via the remote-playback-modify-response message.
+
+: remoting-video
```
Comment (baylesj) [inline index.bs]: Can this be omitted if we only support audio?

File: index.bs [hunk 3]
```diff
@@ -902,13 +906,29 @@ remote-playback-modify-request message with the following values:
...
+:: Videos encodings that the controller can send to the receiver once they
+    are acked via the remote-playback-modify-response message.
+
+: remoting-stats-interval
```
Comment (baylesj) [inline index.bs]: Looks like from the CDDL below that this is optional?

File: index.bs (hunk 1 above)
Comment (chrisn) [reply index.bs]: Remote playback of audio streams is a use case we'd like to see supported. I hope there is implementer interest!

**Pull request merged**